rtloader/common/builtins/datadog_agent.c

```diff
@@ -289,6 +297,32 @@ PyObject *get_clustername(PyObject *self, PyObject *args)
     Py_RETURN_NONE;
 }
 
+/*! \fn PyObject *get_tracemalloc_enabled(PyObject *self, PyObject *args)
+    \brief This function implements the `datadog-agent.get_tracemalloc_enabled` method, returning
+    whether or not tracemalloc is enabled
+    \return a PyObject * pointer to Py_True or Py_False with the state of tracemalloc.
+
+    This function is callable as the `datadog_agent.get_tracemalloc_enabled` python
+    method, it uses the `cb_get_tracemalloc_enabled()` callback to retrieve the value from the agent
+    with CGO. If the callback has not been set `None` will be returned.
+*/
+PyObject *get_tracemalloc_enabled(PyObject *self, PyObject *args)
+{
+    // callback must be set
+    if (cb_get_tracemalloc_enabled == NULL) {
+        Py_RETURN_NONE;
```

Comment:
Should we not return `False` here ?

Reply:
We return `None` everywhere else when the callback hasn't been enabled. But it's a legit point. `None` is falsey too, but I think you're right 👌